### PR TITLE
new cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
new cal csvs, plus some cruise-specific ADCP cal csvs (for Papa/Irm).
Note for the DOSTA-Ds: these are new cal csvs for in-house 2pt cals.  In Vault there are zip files with all associated output files from the in-house cals.  There were some naming format errors in the output files.  Only the 'overall' zip folder names were corrected in Vault.  The cal cert pdf is all that we are releasing to Alfresco so that is the source file & the filenames were corrected before releasing. 

*there was also one old ADCP vendor refurb cal that we didn't have a cal csv for so I added that as well.

### CODE REVIEW

File: calibration/ADCPSL/CGINS-ADCPSL-20503__20240602.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20503,CC_scale_factor1,0.45,constant; GI03FLMA-00011
+20503,CC_scale_factor2,0.45,constant
+20503,CC_scale_factor3,0.45,constant
+20503,CC_scale_factor4,0.45,constant
+20503,CC_bin_size,1000,in cm
+20503,CC_dist_first_bin,1647,in cm
+20503,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cooleyky) [inline calibration/ADCPSL/CGINS-ADCPSL-20503__20240602.csv:8]: looks good

File: calibration/ADCPSN/CGINS-ADCPSN-21717__20161028.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+21717,CC_scale_factor1,0.45,constant; source_file ADCPS-N_WHLS75-1500-I_SN_21717_Configuration_Summary_2016-10-28.pdf
+21717,CC_scale_factor2,0.45,constant
+21717,CC_scale_factor3,0.45,constant
+21717,CC_scale_factor4,0.45,constant
+21717,CC_bin_size,0,in cm; return from vendor - not allocated yet
+21717,CC_dist_first_bin,0,in cm; return from vendor - not allocated yet
+21717,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cooleyky) [inline calibration/ADCPSN/CGINS-ADCPSN-21717__20161028.csv:8]: looks good

File: calibration/ADCPSN/CGINS-ADCPSN-21717__20240602.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+21717,CC_scale_factor1,0.45,constant; GI01SUMO-00011
+21717,CC_scale_factor2,0.45,constant
+21717,CC_scale_factor3,0.45,constant
+21717,CC_scale_factor4,0.45,constant
+21717,CC_bin_size,1000,in cm
+21717,CC_dist_first_bin,1647,in cm
+21717,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cooleyky) [inline calibration/ADCPSN/CGINS-ADCPSN-21717__20240602.csv:8]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00375__20240201.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+375,CC_conc_coef,"[-2.555018E+00, 1.022510E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_375_Calibration_Certificate_2024-02-01.pdf"
+375,CC_csv,"[2.63487E-03, 1.15964E-04, 2.04419E-06, 1.73723E02, -2.57377E-01, -4.19838E01, 3.45392E00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00375__20240201.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00392__20240201.csv [hunk 5]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+392,CC_conc_coef,"[-1.287979E+00, 1.007467E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_392_Calibration_Certificate_2024-02-01.pdf"
+392,CC_csv,"[2.60978E-03, 1.11076E-04, 2.20207E-06, 1.85694E02, -2.76272E-01, -4.69558E01, 3.70203E00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00392__20240201.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00427__20240207.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+427,CC_conc_coef,"[-9.850448E-01, 1.010997E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_427_Calibration_Certificate_2024-02-07.pdf"
+427,CC_csv,"[2.58444E-03, 1.08688E-04, 2.20241E-06, 2.28620E02, -3.14344E-01, -5.54698E01, 4.59266E00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00427__20240207.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00439__20240130.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+439,CC_conc_coef,"[-5.905477E+00, 1.032557E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_439_Calibration_Certificate_2024-01-30.pdf"
+439,CC_csv,"[2.69089E-03, 1.12155E-04, 2.34900E-06, 1.85306E02, -2.28389E-01, -4.54668E01, 3.64945E00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00439__20240130.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00457__20240202.csv [hunk 8]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+457,CC_conc_coef,"[-3.690186E+00, 1.028660E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_457_Calibration_Certificate_2024-02-02.pdf"
+457,CC_csv,"[2.55523E-03, 1.07579E-04, 2.07549E-06, 1.65615E02, -2.35507E-01, -3.71105E01, 3.26954E00]",
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00457__20240202.csv:3]: looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20240417.csv [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+ACS-130,CC_tbins,"[0.740340, 1.304964, 2.421190, 3.436538, 4.471500, 5.518889, 6.473333, 7.505833, 8.496522, 9.471304, 10.485600, 11.493043, 12.491667, 13.506250, 14.508696, 15.498800, 16.486522, 17.486400, 18.494800, 19.479583, 20.491200, 21.486000, 22.488000, 23.475600, 24.484167, 25.507200, 26.499231, 27.501600, 28.503600, 29.497200, 30.505000, 31.497333, 32.490000, 33.511379, 34.485000]",
+ACS-130,CC_taarray,SheetRef:CC_taarray,
+ACS-130,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20240417.csv:9]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20240417__CC_taarray.ext [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+0.004445,0.004123,0.003634,0.003693,0.003666,0.003307,0.003416,0.003275,0.002716,0.002737,0.002973,0.002820,0.002501,0.002212,0.002280,0.001940,0.001576,0.001352,0.001304,0.001195,0.000794,0.000575,0.000539,0.000410,0.000339,0.000000,0.000219,-0.000126,-0.000102,-0.000393,-0.000563,-0.000710,-0.000776,-0.001097,-0.001300
+0.004312,0.003980,0.003617,0.003757,0.003587,0.003277,0.003457,0.003316,0.002732,0.002815,0.003007,0.002796,0.002628,0.002396,0.002346,0.001971,0.001564,0.001370,0.001402,0.001306,0.000844,0.000681,0.000692,0.000556,0.000336,0.000000,0.000269,-0.000040,-0.000162,-0.000365,-0.000469,-0.000657,-0.000722,-0.001129,-0.001246
+0.004543,0.004149,0.003708,0.003786,0.003653,0.003507,0.003484,0.003454,0.002826,0.002825,0.003089,0.002851,0.002666,0.002439,0.002487,0.002045,0.001584,0.001363,0.001420,0.001381,0.000802,0.000630,0.000667,0.000450,0.000395,0.000000,0.000277,-0.000007,-0.000079,-0.000251,-0.000382,-0.000607,-0.000660,-0.001060,-0.001156
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20240417__CC_taarray.ext:82]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20240417__CC_tcarray.ext [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+0.004229,0.005323,0.005263,0.005075,0.004964,0.004687,0.003963,0.003721,0.003659,0.003283,0.003053,0.002975,0.002795,0.002465,0.002465,0.002240,0.001886,0.001697,0.001420,0.001303,0.001054,0.000965,0.000680,0.000387,0.000203,0.000000,-0.000279,-0.000367,-0.000535,-0.000607,-0.000786,-0.001110,-0.001527,-0.001484,-0.001542
+0.002825,0.003992,0.004334,0.004398,0.004465,0.004048,0.003434,0.003174,0.002998,0.002973,0.002609,0.002665,0.002409,0.002151,0.002070,0.002132,0.001783,0.001798,0.001236,0.001193,0.001032,0.000898,0.000762,0.000472,0.000145,0.000000,-0.000295,-0.000446,-0.000558,-0.000645,-0.000864,-0.001313,-0.001370,-0.001857,-0.002000
+0.001422,0.002764,0.003236,0.003340,0.003355,0.003216,0.002769,0.002538,0.002447,0.002369,0.002146,0.002168,0.001956,0.001873,0.001824,0.001769,0.001528,0.001282,0.001196,0.000955,0.000921,0.000872,0.000616,0.000421,0.000268,0.000000,-0.000132,-0.000372,-0.000744,-0.000877,-0.001131,-0.001546,-0.001969,-0.002124,-0.002372
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20240417__CC_tcarray.ext:82]: spot checked, looks good

**Review by cooleyky — APPROVED** [review]
looks good

File: calibration/ADCPSL/CGINS-ADCPSL-18261__20240528.csv [hunk 12]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+18261,CC_scale_factor1,0.45,constant; GP03FLMA-00011
+18261,CC_scale_factor2,0.45,constant
+18261,CC_scale_factor3,0.45,constant
+18261,CC_scale_factor4,0.45,constant
+18261,CC_bin_size,1000,in cm
+18261,CC_dist_first_bin,1647,in cm
+18261,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-18261__20240528.csv:8]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-20502__20240528.csv [hunk 13]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20502,CC_scale_factor1,0.45,constant; GP03FLMB-00011
+20502,CC_scale_factor2,0.45,constant
+20502,CC_scale_factor3,0.45,constant
+20502,CC_scale_factor4,0.45,constant
+20502,CC_bin_size,1000,in cm
+20502,CC_dist_first_bin,1647,in cm
+20502,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-20502__20240528.csv:8]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-20503__20240602.csv (hunk 1 above)
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-20503__20240602.csv:8]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-22650__20240602.csv [hunk 14]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+22650,CC_scale_factor1,0.45,constant; GI03FLMB-00011
+22650,CC_scale_factor2,0.45,constant
+22650,CC_scale_factor3,0.45,constant
+22650,CC_scale_factor4,0.45,constant
+22650,CC_bin_size,1000,in cm
+22650,CC_dist_first_bin,1647,in cm
+22650,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-22650__20240602.csv:8]: looks good

File: calibration/ADCPSN/CGINS-ADCPSN-21717__20161028.csv (hunk 2 above)
Comment (cdobs) [inline calibration/ADCPSN/CGINS-ADCPSN-21717__20161028.csv:8]: looks good

File: calibration/ADCPSN/CGINS-ADCPSN-21717__20240602.csv (hunk 3 above)
Comment (cdobs) [inline calibration/ADCPSN/CGINS-ADCPSN-21717__20240602.csv:8]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00375__20240201.csv (hunk 4 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00375__20240201.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00392__20240201.csv (hunk 5 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00392__20240201.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00427__20240207.csv (hunk 6 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00427__20240207.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00439__20240130.csv (hunk 7 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00439__20240130.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00457__20240202.csv (hunk 8 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00457__20240202.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00460__20240117.csv [hunk 15]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+460,CC_conc_coef,"[-1.831417E+00, 1.035129E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_460_Calibration_Certificate_2024-01-17.pdf"
+460,CC_csv,"[2.76820E-03, 1.15058E-04, 2.40587E-06, 1.52335E02, -2.15616E-01, -3.79270E01, 3.05961E00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00460__20240117.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00462__20240130.csv [hunk 16]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+462,CC_conc_coef,"[-4.021958E+00, 1.034117E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_462_Calibration_Certificate_2024-01-30.pdf"
+462,CC_csv,"[2.60585E-03, 1.10012E-04, 2.25393E-06, 1.89243E02, -2.40919E-01, -4.65361E01, 3.73760E00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00462__20240130.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00507__20240202.csv [hunk 17]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+507,CC_conc_coef,"[-8.383415E-01, 1.007819E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_507_Calibration_Certificate_2024-02-02.pdf"
+507,CC_csv,"[2.67179E-03, 1.11514E-04, 2.33518E-06, 1.79519E02, -2.49093E-01, -4.81798E01, 3.63603E00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00507__20240202.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00512__20240206.csv [hunk 18]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+512,CC_conc_coef,"[-1.095398E+00, 1.008879E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_512_Calibration_Certificate_2024-02-06.pdf"
+512,CC_csv,"[2.66393E-03, 1.13860E-04, 2.27847E-06, 1.87276E02, -2.47635E-01, -5.21275E01, 3.70112E00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00512__20240206.csv:3]: looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20240417.csv (hunk 9 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20240417.csv:9]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20240417__CC_taarray.ext (hunk 10 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20240417__CC_taarray.ext:82]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00130__20240417__CC_tcarray.ext (hunk 11 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00130__20240417__CC_tcarray.ext:82]: spot checked, looks good

File: calibration/PHSENE/CGINS-PHSENE-P0138__20240321.csv [hunk 19]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+P0138,CC_ea434,17372.0, source_file PHSEN-E_SAMI-pH_SN_P0138_Validation_Files_2024-03-21.zip > SAMI_P0138_210324t_ValCert.pdf
+P0138,CC_ea578,94.1,
+P0138,CC_eb434,2284.1,
+P0138,CC_eb578,38676.5,
+P0138,CC_ind_off,0,
+P0138,CC_ind_slp,1,
+P0138,CC_psal,35,Constant
+P0138,CC_sami_bits,12,
```
Comment (cdobs) [inline calibration/PHSENE/CGINS-PHSENE-P0138__20240321.csv:9]: looks good

**Review by cdobs — APPROVED** [review]
Everything looks good

**Pull request merged**